### PR TITLE
Mask each line of multi-line secrets

### LINKED ISSUE #205: Exposes secrets in plaintext
When using v0.2.1 actions output secrets in plaintext, here's an example

pipeline output
```go
Run 
  echo "-----BEGIN OPENSSH PRIVATE KEY-----"
  
```

pipeline itself
```go
on: push
jobs:
  test:
    strategy:
      matrix:
        go-version: [1.16.x]
    runs-on: self-hosted
    steps:
      - name: Import Secrets
        id: secrets
        uses: github/vault-action@v2.1.2
        with:
          url: https://vault:8200
          method: approle
          roleId: abc
          secretId: ${{ secrets.SECRETID }}
          tlsSkipVerify: true
          secrets: |
            secret/data/secret ssh-key | SSH_KEY ;
      - name: Install Go
        uses: actions/setup-go@v2
        with:
          go-version: ${{ matrix.go-version }}
      - name: Checkout code
        uses: actions/checkout@v2
      - name: Setup bitbucket
        run: |
          echo "${{ env.SSH_KEY }}" > ~/.ssh/id_rsa
      - name: Test
        run: make test

```

### PR DESCRIPTION
**Pull request by tomhjp**
Closes #205.

When fetching multi-line secrets from Vault, the secret was not being properly registered with GH actions for masking in the output logs. See the linked issue for full details - the fix itself is fairly trivial.

Still todo:

- [x] Run some real infrastructure tests to confirm the fix
- ~[ ] Determine whether the large `index.js` diff is problematic, looks like I probably have some different local versions installed to the previous builder~

### CODE REVIEW

**Comment by tvoran** [comment]
FWIW, I think we usually wait to commit dist/index.js until a release.

**Review by jasonodonnell — COMMENTED** [review]
LGTM, but @tvoran is right, please remove the compiled `index.js` changes and we'll do that when we're ready to release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+Bugs:
```
Comment (jasonodonnell) [inline CHANGELOG.md:3]:
```suggestion
Security:
```

**Review by jasonodonnell — APPROVED** [review]
LGTM

**Review by tvoran — APPROVED** [review]
Seems to work for me 👍

File: src/action.js [hunk 2]
```diff
@@ -77,8 +77,12 @@ async function exportSecrets() {
...
         if (cachedResponse) {
             core.debug('ℹ using cached response');
-        }        
-        command.issue('add-mask', value);
+        }
+        for (const line of value.replace(/\r/g, '').split('\n')) {
+            if (line.length > 0) {
+                command.issue('add-mask', line);
+            }
+        }
```
[the remark this answers is not in the document]
Comment (tomhjp) [reply src/action.js:85]: Thanks for testing that, good to know. Unless you have any objections, I'll leave this as-is though just to remain on the defensive side.

**Pull request merged**